src/main/java/org/mvc/controllers/MessagesController.java

```diff
@@ -0,0 +1,43 @@
+package org.mvc.controllers;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+
+import org.mvc.handlers.MessagesHandler;
+import org.mvc.models.IncomingMessage;
+import org.mvc.models.SignIn;
+
+@RestController
+public class MessagesController {
+    ModelAndView mav = new ModelAndView();
+
+    @PostMapping("/signin")
+    public ModelAndView signIn(SignIn signIn) {
+        mav.clear();
+        mav.setViewName("messages");
+        mav.addObject("user", signIn.getUser());
```

Comment:
Aquí hay un detalle, cuando se usan dos usuarios distintos, al final solo se marcan mensajes como enviados por el ultimo que ha hecho sign-in. Es decir:
Usuario A hace sign in
Usuario A manda un mensaje
Usuario B hace sign in
Usuario B manda un mensaje
Usuario A manda un mensaje (este mensaje queda registrado como si fuera del usuario A, incluso en la ventana de los mensajes, el user se actualiza al del Usuario B)

Reply:
Para arreglar este bug, utilicé `cookies`